### ISSUE TITLE
Requesting OCSP response from server

### ISSUE THREAD

**Issue by schmidt-marvin** [wdawson/revoker#1]
First off, thanks for the excellent app. I'm pretty new to certificates, OCSP and so on, which is why I chose this implementation for testing purposes. I do have a suggestion for the introduction README:

When I started working with the OCSP Responder, I specified the needed files and ports within the .yml-config. I worked with a full test set of certificates, including three parties requiring OCSP. For each one, I specified the URL to the OCSP service within the respective X509v3 extension fields. Following the instructions on the README, I specified each field as ``http://localhost:10123``, for example. Upon placing my OCSP requests to this URL, I kept on getting the following response every time:
```
{"code":404,"message":"HTTP 404 Not Found"}
````
After some hours of head scratching, I found out from your source code, that OCSP requests are supposed to be sent to a domain formatted like this: ``http://localhost:10123/ocsp``. Thankfully, this allowed the actual requests to be made to your app and communicated back to my validator.

It'd be great if the domain formatting could be made clearer in the README.

**Comment by wdawson** [comment]
Hey @schmidt-marvin, I'm sorry I never got a notification for this. If you still remember what might have helped, I'd happily take a PR. It seems like you'd want clarity that the OCSP responder has a particular path on the server?

**Comment by schmidt-marvin** [comment]
Hi @wdawson! It's all good, thanks for getting back to me. I've since completed my work on the uni project I've needed this for. If I remember correctly, I was merely a bit confused about how to use the app, and my suggestion was to add some hints to the README.md. I've added some functionality within it, as well as an improved README within the forked repository, but frankly I'd have to review my changes before requesting a merge. Also, as a student, I'm kind of a git novice and probably won't comply with best practices in doing this.

I have however marked the additions to your README back then. Whenever you work with this again in the future, feel free to use them, they're marked with an asterisk: https://github.com/schmidt-marvin/revoker
